src/api/store/userprofile.py

```diff
@@ -205,20 +205,21 @@ def create_user(self, context: Context, args) -> (dict, MassEnergizeAPIError):
 
       if not email:
         return None, CustomMassenergizeError("email required for sign up")
-      
       user = UserProfile.objects.filter(email=email).first()
+      print("this is the user")
+      print(user)
       if not user:
         new_user: UserProfile = UserProfile.objects.create(
           full_name = args.get('full_name'), 
           preferred_name = args.get('preferred_name', None), 
           email = args.get('email'), 
           is_vendor = args.get('is_vendor', False), 
-          accepts_terms_and_conditions = args.pop('accepts_terms_and_conditions', False)
+          accepts_terms_and_conditions = args.pop('accepts_terms_and_conditions', False), 
+          preferences = {'color': args.get('color')}
```

Comment:
only save the color if specified as a parameter.
In the front end, if the color isn't specified, and the profile  picture isn't uploaded, let's prompt them (once) to choose a color or invite to upload a profile pic